### PR TITLE
Remove source files of dlib

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: installer/targets/openface/dlib.bash [hunk 1]
```diff
@@ -29,6 +29,9 @@ sudo sh -c 'echo "/usr/lib/openblas-base/" > /etc/ld.so.conf.d/libopenblas-base.
 #Finally, run ldconfig to update the cache.
 sudo ldconfig
 
+#Remove source files
+sudo rm -rf ~/src
```
Comment (Rayman) [inline installer/targets/openface/dlib.bash]: I put a lot of my own programs in `~/src` so this is obviously wrong.

Comment (MatthijsBurgh) [reply installer/targets/openface/dlib.bash]: Why not dowload the stuff in /tmp? The binaries are copied and the other files aren't used anymore, right?

**Comment by LoyVanBeek** [comment]
Why delete the source files? I don't know how often we recompile this stuff. 

I'd rather have redistributeable binaries for the whole thing.

**Comment by LoyVanBeek** [comment]
Ugh, this is a compile and forget package, if you want to recompile you have bigger issues I suppose.

**Pull request merged**